doc/en/CAS/Simplification.md

```diff
@@ -52,17 +52,45 @@ This includes the answer tests as needed.
 The level of simplification performed by Maxima can be controlled by changing Maxima's global variable `simp`, e.g.
 
     simp:true
+    
+When `simp` is set to `false`, no simplification is performed and Maxima is quite happy to deal with an expression such as \(1+4\) without actually performing the addition.
+This is most useful for dealing with very elementary expressions, and for [showing working](../CAS/Matrix/#Showing_working).
 
-This variable can be set at the question level using the [options](../Authoring/Options.md) or for each [Potential response tree](../Authoring/Potential_response_trees.md). Within [CASText](../Authoring/CASText.md), the setting can be controlled by appending `simp=...`, for example: 
+This variable can be set at the question level using the [options](../Authoring/Options.md) or for each [Potential response tree](../Authoring/Potential_response_trees.md).
 
-    {@3/9,simp=false@}
+When `simp` is set to `false`, you can evaluate an expression with simplifcation turned on by using `ev(..., simp)`, for example:
+
+    simp:false;
+    a:ev(1+1,simp);
+    
+will give \(a=2\).
+
+### Within CASText (question text, general feedback, etc.)
+
+Sometimes it is useful to control the level of simplification applied to expressions included within [CASText](../Authoring/CASText.md) using `{@...@}`.
+In particular, to show steps in working, it is often necessary to turn simplification off.
+
+Note that it is not possible to reliably control the value of `simp` within the CAS expression itself: for instance, `{@(simp:false,1+1)@}` will display as \(2\) if `simp` is set to `true` at the question level.
+(This is because of the way STACK assembles all the CASText code into a single Maxima call.)
```

Comment:
Hopefully, this will be fixed for this simpler case for 4.4.2. At this moment, we are having a problem detecting the simplification setting in use in this form of expression, but we'll see if we can fix that.

While we do indeed turn the whole thing into a single expression, we can control simplification inside that expression we just need to detect when we need to do so.